_posts/2017-06-09-Principles-of-object-oriented-design.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+author: domenique
+disqus: false
+header-img: "img/patterns.jpg"
+---
+
+_SOLID_, The famous acronym coined by Micheal Feathers describing the first five principles of object oriented design composed by Robert C. Martin a.k.a Uncle Bob. What most people tend to forget is that __SOLID describes only five of the in total eleven principles of object oriented design__. These principles date all the way back to 1995, yes Uncle Bob is really that old :)
+
+This post will briefly describe the well known SOLID principles, after which we will dive deeper into the 6 other, lesser known principles. This post will ultimately become a series, so do check back for follow-up articles.
+
+# SOLID Principles
+
+The SOLID principles are the first five principles of object oriented design and focus around class design.
```

Comment:
"The first five principles?" OO bestond voor dat deze principes bestonden. Ik zou wat meer kadering geven van waar ze komen en wat hun plaats is. OO is door andere zaken gedefinieerd. Hoe verhouden de SOLID principes zich hiertoe?